app/assets/javascripts/activeadmin_addons/select2.js

```diff
@@ -149,7 +149,7 @@ $(function() {
       if (!!parent) {
         var parentSelector = '#' + model + '_' + parent;
 
-        $(parentSelector).on('change', function(e) {
+        $(el).parent().find(parentSelector).on('change', function(e) {
```

Comment:
Yes. Because when it's multiple nested_select on page, it can't find parent element.